src/promptflow/promptflow/contracts/run_info.py

```diff
@@ -50,7 +100,54 @@ class RunInfo:
 
 @dataclass
 class FlowRunInfo:
-    run_id: str  # flow_run_id:child_flow_run_id
+    """A dataclass representing the run information.
+
+    :param run_id: The id of the run, which equals flow_run_id:child_flow_run_id
```

Comment:
can enclose `flow_run_id:child_flow_run_id` with double backticks to highlight:
![image](https://github.com/microsoft/promptflow/assets/2572521/3024df9b-0bdd-4311-9534-c1772c5f55ab)